setup.py

```diff
@@ -52,7 +52,7 @@
         'numpy>=1.15',
         'psutil',
         'pyradiosky>=0.1.2',
-        'pyuvdata>=2.2.8',
+        'pyuvdata>=2.2.10',
```

Comment:
This requirement change needs to be updated in the readme as well.

Reply:
It also needs to be updated in our developer conda yaml, `environment.yml` in the top level directory.

Reply:
Done.